### PR TITLE
Newsroom Ansible staging server playbook

### PR DESCRIPTION
**Pull request by kaylaziegler**
Story: https://trello.com/c/Cia8Qjhe

This also switches the project to using encrypted secrets. Refer to REAME for new setup instructions.

Staging URL is https://news-staging.chapman.edu

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,9 +15,9 @@ Chapman University's news and stories website.
     bundle install
 
     # Copy dist versions of config files into place
-    cp -v config/secrets.yml{.dist,}
     cp -v config/database.yml{.dist,}
 
+The master encryption key can be found in Passpack. Place the key under `config/` in a new file called `secrets.yml.key`. You will need this key to run Newsroom locally and to run tests.
```
Comment (tatwell) [inline README.md]: One thing to keep in mind is idea that this is a open source project that, for example, other schools might use. Not our first priority but where possible let's see if we can keep it from being too closely coupled to Chapman.

So maybe add a link here to Rails docs on master encryption key and add:

> The master encryption key for Chapman can be found in Passpack.

File: config/secrets.yml.dist [hunk 2]
```diff
@@ -1,32 +0,0 @@
-# Be sure to restart your server when you modify this file.
```
Comment (tatwell) [inline config/secrets.yml.dist]: Why did we get rid of this file?

File: config/secrets.yml.enc [hunk 3]
```diff
@@ -0,0 +1 @@
+ST1i+eW23e3SzCpNrBC5zjnRdNF0RT3+MFI5C6Fz3/Q3aFl6XM30iW5FK9fo4uYs6bCi1WYuafp/KmgnLL66EfkYxHZmDZU3vYWfm6vcdGKs2rs3dOmmfawvNt5Gdb0wifXZD0iMhOIg4svrvfcz7WWlNV0Twzly6WcS6rX5TCtlCX5tFEUxjuKbzwPGKfXCZpUr994iEMKmQcljeQfFojyAg7YWOA/Pb4UTacbH1hBajdPrTq/T3u8Z56jNBhnsqXkD8HxEQa86qK+9TNJKvul7N9hMddKvZismKZtuvK/9hiPj2oh2lZuaHWoXuoypvmXeIMvN8WSKcZVNWlL7ztrPc9y4e5SmZMG40eh43kgwfLHoDltPymoVVscc09PK5Dtnz1muWOvS+12nTsT8QdAu/uRu0LOpD613U4b8Fiyu9E9kkwNKxG1c46WXRVINPcArWM/Fnh53IxI/+Xjx65N3aLyjufw/uMDdR/CkqLSxOECuEH3w1oQDY1a7MTdk92qMDQq4uWrU7wjl3BhmJym4f/uJjvOJKuQYnrlVzYJMCeQakekHS8IyDwYnMs+9rppL1WyCdOYZIuiwaZL+lyI7PYIP8O70D3vgkqWsWtmEOktlVD3Sl85STn3slxwQc+sqQGI+HyW3zOVoVOolYj/+G6pD2wDAsXpy2LtkyGjxeGihgTXuJ6Us2/lb9CfrHN7EsEuqiOEGiCDD6WqiUeIc/6BiNBDaoUrzJedjPZB1QbUbo1pwDML78IrQ6bi1nuvd/LPtLdW56wgwJh4myYOW5ui1/pAIxR/6vU1LI32Qb1tJ/ODfDwLCdNoNyj1A4aJ2GKaAA/FNUFi35TqC7TJAqycG6cFW28LH2ZOLbADFsirKUunENoPHVY4/i1O9g7jecdDZB1IdKxctx14XooaqgFXhLxuknfjUng==--4sOmXoDZKaNBU4LH--cL5cCnHTobuAsDnKms2nlg==
```
Comment (tatwell) [inline config/secrets.yml.enc]: Is this supposed to be in here? Is this the public key?

File: config/secrets.yml.dist (hunk 2 above)
Comment (kaylaziegler) [reply config/secrets.yml.dist]: After switching to encrypted secrets, this file is essentially useless. I'll respond to your other [comment](https://github.com/chapmanu/newsroom/pull/3#discussion_r160767111) with some documentation that explains how the encrypted secrets work.

File: config/secrets.yml.enc (hunk 3 above)
Comment (kaylaziegler) [reply config/secrets.yml.enc]: This is the "secrets" file technically, just encrypted. It's virtually meaningless without the master key (which is _not_ kept in the repository). Rails has a little [documentation](http://edgeguides.rubyonrails.org/5_1_release_notes.html#encrypted-secrets) on the addition of encrypted secrets, plus the link to the pull request that introduced it, but I used this [other article](https://www.engineyard.com/blog/encrypted-rails-secrets-on-rails-5.1#encrypted-secrets) that explains it in clearer detail.

**Pull request merged**